scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -83,11 +72,12 @@ class ScanCode(
     name: String,
     config: ScannerConfiguration
 ) : LocalScanner(name, config) {
-    class Factory : AbstractScannerFactory<ScanCode>("ScanCode") {
+    class Factory : AbstractScannerFactory<ScanCode>(SCANNER_NAME) {
         override fun create(config: ScannerConfiguration) = ScanCode(scannerName, config)
     }
 
     companion object {
+        const val SCANNER_NAME = "ScanCode"
```

Comment:
Nit: Blank line after this one to separate from `private` constants.

Reply:
Done